pyramid_oereb/lib/records/theme.py

```diff
@@ -18,3 +18,8 @@ def __init__(self, code, text):
 
         self.code = code
         self.text = text
+
+    def __str__(self):
+        return '<%s -- code: %s text: %s>' % (
```

Comment:
Iam more the `'hallo {0}'.format('du')` guy. Its a suggestion only ;)

Reply:
I like that better too, as well -> changed.

Reply:
thanks ;)